members/state_il_gov_bruce_rauner.yaml

```diff
@@ -6,115 +6,83 @@ contact_form:
     - fill_in:
         - name: firstName
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_First_x0020_Name"
+          selector: input#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_First_x0020_Name
           value: $NAME_FIRST
           required: true
         - name: lastName
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Last_x0020_Name"
+          selector: input#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Last_x0020_Name
           value: $NAME_LAST
           required: true
         - name: phone
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Phone"
+          selector: input#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Phone
           value: $PHONE
           required: true
         - name: street1
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Address"
+          selector: input#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Address
           value: $ADDRESS_STREET
           required: true
         - name: city
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_City"
+          selector: input#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_City
           value: $ADDRESS_CITY
           required: true
         - name: zip5
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Zip"
+          selector: input#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Zip
           value: $ADDRESS_ZIP5
           required: true
         - name: email
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Email"
+          selector: input#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Email
           value: $EMAIL
           required: true
-        - name: message
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Comment"
+        - name: comment
+          selector: textarea#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Comment
           value: $MESSAGE
           required: true
     - select:
         - name: prefix
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Prefix_x0020_Name"
+          selector: select#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Prefix_x0020_Name
           value: $NAME_PREFIX
           options:
             "Mr.": "Mr."
             "Mrs.": "Mrs."
             "Ms.": "Ms."
         - name: position
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Position"
+          selector: select#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Position
           value: "None"
           required: true
           options:
             - Support
             - Oppose
             - None
-        - name: topic
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Subject"
-          value: $TOPIC
+        - name: subject
+          selector: select#ctl00_ctl40_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_Subject
+          value: $SUBJECT
```

Comment:
subject is an input, topic is for a dropdown with different mapped options. which does the form have?